x-pack/docs/en/security/securing-communications/setting-up-ssl.asciidoc

```diff
@@ -19,21 +18,26 @@ applicable to your cluster.
 certificate and enable TLS on the transport layer. You can also optionally
 enable TLS on the HTTP layer. See
 {ref}/configuring-tls.html#tls-transport[Encrypting Communications Between Nodes in a Cluster] and
-{ref}/configuring-tls.html#tls-http[Encrypting HTTP Client Communications]. 
+{ref}/configuring-tls.html#tls-http[Encrypting HTTP Client Communications].
 
-. Configure the {monitor-features} to use encrypted connections. See <<secure-monitoring>>.
++
+NOTE: {ref}/configuring-tls.html#tls-http[Encrypting HTTP Client Communications] is strongly recommended if you want to enable TLS encryption for Kibana, Logstash and Beats.
```

Comment:
Encrypting HTTP Client Communications is **required** if you want to enable TLS ....